### ISSUE TITLE
Start point of vowels U and UU with consonant PHA ಫು಼ ಫೂ಼ in Noto Sans Kannada

### ISSUE THREAD

**Issue by YusukeShirakihara** [notofonts/kannada#52]
Hello. Thank you for providing us with great fonts.
I’d like to report what a native Kannada speaker hopes to be updates in Noto Sans Kannada.

It applies to all Noto Sans Kannada fonts:
https://github.com/notofonts/kannada/releases/download/NotoSansKannada-v2.004/NotoSansKannada-v2.004.zip

ಫು಼
U+0CAB ಫ
U+0CC1 ು
U+0CBC ಼

ಫೂ಼
U+0CAB ಫ
U+0CC2 ೂ
U+0CBC ಼

As for the two characters above , it is better for the vowels U and UU to start beneath the consonant PHA. 
![issue1_1_NotoSansKannada-Regular ttf](https://github.com/notofonts/kannada/assets/122098208/ab9ef80c-baad-4cbd-9a96-a5a91e16c3eb)

In other words, it is hoped to start from the same point, in case these characters do not have the Nukta like ಫು and ಫೂ.
![issue1_2_NotoSansKannada-Regular ttf](https://github.com/notofonts/kannada/assets/122098208/c9ff612a-0e70-400e-abf2-d1fd0b0bece2)

**Comment by YusukeShirakihara** [comment]
Since helpful comments were posted on the following similar topic, this topic can be closed. Thank you.
https://github.com/notofonts/kannada/issues/56

**Comment by YusukeShirakihara** [comment]
Since helpful comments were posted on the following similar topic, this topic can be closed. Thank you.
https://github.com/notofonts/kannada/issues/56
